ui/src/alfresco/DockerContainerCreate.tsx

```diff
@@ -111,27 +134,37 @@ const FeedbackPanel = ({ alfrescoState }) => {
     message = 'Stopping Alfresco containers...';
     actionInProgress = true;
   }
+  if (isReady(alfrescoState)) {
+    message =
+      "Alfresco is Ready! Click the 'Open in browser' link and use the default credentials admin/admin";
```

Comment:
We may add this as i18n resource, despite we're accepting only English as language currently.

Reply:
Opened an issue to refactor the messaging.